ansible/group_vars/all

```diff
@@ -1,4 +1,6 @@
 ansible_python_interpreter: '/usr/bin/python3'
+pulp_scl_enable_python3: '{% if ansible_distribution == "RedHat" %}/usr/bin/scl enable rh-python35 -- {% endif %}'
```

Comment:
Please use the python36 SCL - AFAIK the only reason we were targeting 3.5 is that an SCL for 3.6 didn't exist until a few weeks ago.